packages/start-plugin-core/tests/importProtection/sourceLocation.test.ts

```
@@ -1,14 +1,19 @@
 import { describe, expect, test } from 'vitest'
 import {
+  createImportSpecifierLocationIndex,
   ImportLocCache,
```

ESLint

[error] Member 'ImportLocCache' of the import declaration should be sorted alphabetically.

(sort-imports)